portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/ManagePublications.testcase

```diff
@@ -168,6 +168,29 @@ definition {
 			changeUser = "Test Test");
 	}
 
+	@description = "This is a use case for LPS-184911 - Saving Publications settings when it is disabled causes PrincipalException."
+	@priority = 3
+	test CanSavePublicationsSettingsWhenItIsDisabled {
+		Publications.disablePublications();
```

Comment:
Instead of disabling Publications manually, override the configuration
```
property osgi.module.configuration.file.names = "com.liferay.change.tracking.configuration.CTSettingsConfiguration.config";
property osgi.module.configurations = "";
```